StompClientLib/Classes/StompClientLib.swift

```diff
@@ -72,17 +74,28 @@ public class StompClientLib: NSObject, SRWebSocketDelegate {
     var sessionId: String?
     var delegate: StompClientLibDelegate?
     var connectionHeaders: [String: String]?
-    public var connection: Bool = false
+    private(set) var isConnected: Bool = false
```

Comment:
Why did you make it public to private? In some cases, devs might want to check the 'connection' status.

Reply:
Only setter is private, getter is public. (devs can check the status, but they can't change it from outside.)

Reply:
Got it :) Thank you for the explanation.